clients/src/test/java/org/apache/kafka/clients/consumer/KafkaConsumerTest.java

```diff
@@ -425,7 +434,7 @@ public void verifyNoCoordinatorLookupForManualAssignmentWithSeek() {
         client.prepareResponse(listOffsetsResponse(Collections.singletonMap(tp0, 50L)));
         client.prepareResponse(fetchResponse(tp0, 50L, 5));
 
-        ConsumerRecords<String, String> records = consumer.poll(5);
+        ConsumerRecords<String, String> records = consumer.poll(Duration.ofMillis(1));
```

Comment:
time never actually advances because we use `MockTime(0)`, so any timeout at all will block indefinitely and keep polling until we get a response. This only works in tests that expect a response, otherwise it would loop forever.